modules/apps/configuration-admin/configuration-admin-api/src/main/java/com/liferay/configuration/admin/configuration/InstanceSettingsClientExtensionsConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.configuration.admin.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Anderson Luiz
+ */
+@ExtendedObjectClassDefinition(
+	category = "instance-configuration", generateUI = true,
+	scope = ExtendedObjectClassDefinition.Scope.COMPANY
+)
+@Meta.OCD(
+	id = "com.liferay.configuration.admin.configuration.InstanceSettingsClientExtensionsConfiguration",
+	localization = "content/Language",
+	name = "instance-settings-client-extensions-configuration-name"
+)
+public interface InstanceSettingsClientExtensionsConfiguration {
+
+	@Meta.AD(
+		deflt = "", name = "js-client-extensions-external-reference-codes",
```

Comment:
Since this won't be just the external reference code, but will also include other metadata, this should probably just be called `globalJSCETs`